tensorflow/python/lib/io/file_io_test.py

```diff
@@ -451,6 +451,12 @@ def testReadlines(self):
     lines = f.readlines()
     self.assertSequenceEqual(lines, data)
 
+  def testUTF8StringPath(self):
```

Comment:
I'm not sure if I'm reading [PEP 263](https://www.python.org/dev/peps/pep-0263/) correctly, but I think we'll need `# coding=UTF-8 ` near the top of the file. (IIUC, without such a comment, it'll be treated as ASCII.)